tests/test_scalar.py

```diff
@@ -0,0 +1,125 @@
+import pytest
+import responses
+
+from moztelemetry.scalar import Scalar
+
+
+nightly_file = 'https://hg.mozilla.org/mozilla-central/raw-file/tip/toolkit/components/telemetry/Scalars.yaml'
+release_file = 'https://hg.mozilla.org/releases/mozilla-release/raw-file/tip/toolkit/components/telemetry/Scalars.yaml'
+scalar_str = ''.join(open('tests/Scalars.yaml', 'r').readlines())
+
+
+@pytest.fixture(autouse=True)
```

Comment:
I think this will make this fixture run for every test in the codebase, which makes things a bit unpredictable. Let's include it as a dependency where it's needed. There's an example of how to use responses with pytest here https://github.com/getsentry/responses/pull/118/commits/e03116ce8f25b5c0fc43ff7b7e5fb138cd24dc02